src/instruments/src/EFB/Ground/Pages/Payload/Payload.tsx

```diff
@@ -0,0 +1,871 @@
+    const [paxDDesired, setPaxDDesired] = useSimVar('L:A32NX_PAX_TOTAL_ROWS_22_29_DESIRED', 'number');
+
+    const [stationSize, setStationLen] = useState<number[]>([]);
+    const totalPax = useCallback(() => pax && pax.length > 0 && pax.reduce((a, b) => a + b), [...pax]);
+    const maxPax = useCallback(() => ((stationSize && stationSize.length > 0) ? stationSize.reduce((a, b) => a + b) : -1), [stationSize]);
+
+    const [aFlags, setAFlags] = useBitFlags('PAX_FLAGS_A');
+    const [bFlags, setBFlags] = useBitFlags('PAX_FLAGS_B');
+    const [cFlags, setCFlags] = useBitFlags('PAX_FLAGS_C');
+    const [dFlags, setDFlags] = useBitFlags('PAX_FLAGS_D');
+
+    const paxDesired = [paxADesired, paxBDesired, paxCDesired, paxDDesired];
+    const [setPaxDesired] = useState([setPaxADesired, setPaxBDesired, setPaxCDesired, setPaxDDesired]);
+    const totalPaxDesired = useCallback(() => (paxDesired && paxDesired.length > 0 && paxDesired.reduce((a, b) => parseInt(a) + parseInt(b))), [...paxDesired]);
+
+    const [aFlagsDesired, setAFlagsDesired] = useBitFlags('PAX_FLAGS_A_DESIRED');
+    const [bFlagsDesired, setBFlagsDesired] = useBitFlags('PAX_FLAGS_B_DESIRED');
+    const [cFlagsDesired, setCFlagsDesired] = useBitFlags('PAX_FLAGS_C_DESIRED');
+    const [dFlagsDesired, setDFlagsDesired] = useBitFlags('PAX_FLAGS_D_DESIRED');
+
+    const activeFlags = [aFlags, bFlags, cFlags, dFlags];
+    const desiredFlags = [aFlagsDesired, bFlagsDesired, cFlagsDesired, dFlagsDesired];
+    const setActiveFlags = [setAFlags, setBFlags, setCFlags, setDFlags];
+    const setDesiredFlags = [setAFlagsDesired, setBFlagsDesired, setCFlagsDesired, setDFlagsDesired];
+
+    const [clicked, setClicked] = useState(false);
+
+    const [fwdBag] = useSimVar('L:A32NX_CARGO_FWD_BAGGAGE_CONTAINER', 'Number', 200);
+    const [aftCont] = useSimVar('L:A32NX_CARGO_AFT_CONTAINER', 'Number', 200);
+    const [aftBag] = useSimVar('L:A32NX_CARGO_AFT_BAGGAGE', 'Number', 200);
+    const [aftBulk] = useSimVar('L:A32NX_CARGO_AFT_BULK_LOOSE', 'Number', 200);
+
+    const cargo = [fwdBag, aftCont, aftBag, aftBulk];
+
+    const [fwdBagDesired, setFwdBagDesired] = useSimVar('L:A32NX_CARGO_FWD_BAGGAGE_CONTAINER_DESIRED', 'Number', 200);
+    const [aftContDesired, setAftContDesired] = useSimVar('L:A32NX_CARGO_AFT_CONTAINER_DESIRED', 'Number', 200);
+    const [aftBagDesired, setAftBagDesired] = useSimVar('L:A32NX_CARGO_AFT_BAGGAGE_DESIRED', 'Number', 200);
+    const [aftBulkDesired, setAftBulkDesired] = useSimVar('L:A32NX_CARGO_AFT_BULK_LOOSE_DESIRED', 'Number', 200);
+
+    const cargoDesired = [fwdBagDesired, aftContDesired, aftBagDesired, aftBulkDesired];
+    const setCargoDesired = [setFwdBagDesired, setAftContDesired, setAftBagDesired, setAftBulkDesired];
+    const totalCargoDesired = useCallback(() => ((cargoDesired && cargoDesired.length > 0) ? cargoDesired.reduce((a, b) => parseInt(a) + parseInt(b)) : -1), [...cargoDesired, ...paxDesired]);
```

Comment:
Consider using useMemo since you are more or less just storing the callback to calculate a value which is better suited for useMemo